sdk/cosmos/SessionRecords/create_database_and_collection/1_request.json

```diff
@@ -0,0 +1,77 @@
+{
+  "uri": "https://mindflavor2.documents.azure.com/dbs/sample/colls",
+  "method": "POST",
+  "headers": {
+    "x-ms-version": "2018-12-31",
+    "authorization": "<<STRIPPED>>",
+    "user-agent": "azsdk-rust-cosmos/0.1.0 (1.56.0-nightly; linux; x86_64)",
+    "x-ms-date": "Sat, 11 Sep 2021 12:13:24 GMT"
+  },
+  "body": [
+    123,
+    34,
+    105,
+    100,
+    34,
+    58,
+    34,
+    112,
+    97,
+    110,
+    122,
+    97,
+    100,
+    111,
+    114,
+    111,
+    34,
+    44,
+    34,
+    112,
+    97,
+    114,
+    116,
+    105,
+    116,
+    105,
+    111,
+    110,
+    75,
+    101,
+    121,
+    34,
+    58,
+    123,
+    34,
+    112,
+    97,
+    116,
+    104,
+    115,
+    34,
+    58,
+    91,
+    34,
+    47,
+    105,
+    100,
+    34,
+    93,
+    44,
+    34,
+    107,
+    105,
+    110,
+    100,
+    34,
+    58,
+    34,
+    72,
+    97,
+    115,
+    104,
+    34,
+    125,
+    125
+  ]
```

Comment:
Checking in multiple of these files will cause rather large diffs. Would it be possible to hex-encode before storing these in the file? With only one byte stored per line, even small JSON responses would quickly grow to thousands of lines.

Reply:
Love the idea. We control both serialization and deserialization so it should be easy to add base64 encoding. 👍

Reply:
Done! You were right, it's much cleaner.